### PR TITLE
delete the show

### PR DESCRIPTION
**Pull request by shailja-imw**
=>Show delete API call

### CODE REVIEW

File: app/components/Shows.vue [hunk 1]
```diff
@@ -1,7 +1,20 @@
 <script setup>
 const props = defineProps(['showsData'])
+const config = useRuntimeConfig()
+const base_url = config.public.base_url
 const shows = ref(props.showsData.data)
 const isPending = ref(props.showsData.isPending)
+
+async function deleteShow(id, show) {
+  if (confirm('Are you sure you wanted to remove ' + show + ' this show')) {
```
Comment (monikagorkhiya) [inline app/components/Shows.vue:9]: You can put bootstrap modal

**Review by monikagorkhiya — CHANGES_REQUESTED** [review]

File: app/components/Shows.vue [hunk 2]
```diff
@@ -46,7 +59,10 @@ const isPending = ref(props.showsData.isPending)
                 /></NuxtLink>
                 <img
                   src="~/assets/images/icons/delete.svg"
-                  class="main__table-btn btn_delete"
+                  class="main__table-btn btn_delete cursor-pointer"
+                  data-bs-toggle="modal"
+                  data-bs-target="#exampleModal"
+                  @click="deleteShow(show.id, show.title)"
```
Comment (monikagorkhiya) [inline app/components/Shows.vue:65]: Where this is used?

**Review by monikagorkhiya — CHANGES_REQUESTED** [review]

File: app/components/Shows.vue (hunk 1 above)
Comment (shailja-imw) [reply app/components/Shows.vue:9]: okay

File: app/components/Shows.vue (hunk 2 above)
Comment (shailja-imw) [reply app/components/Shows.vue:65]: I had used the bootstrap modal but then I did with the alert confirm

**Review by monikagorkhiya — APPROVED** [review]

**Pull request merged**